README.rst

```diff
@@ -130,9 +136,9 @@ To build the PHP driver for Windows:
 
        .\\scripts\\setup_php_sdk.bat x64 Release VS16 C:\\php-sdk
 
-#. Download and install the PHP binaries, or build PHP yourself.
+#. Download and install the PHP binaries, or build the PHP source code.
```

Comment:
This one still not accurate. There is no option for customer to choose build or not build the PHP souce code. It's always needed as the driver build needs some artifacts from there. This step is not downloading and install PHP binaries. It's download and build PHP source code.

Reply:
Done